lab-dan/server.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const net = require('net');
+const EE = require('events');
+
+const PORT = process.env.PORT || 3000;
+
+function start (chatClient, cb) {
+  let pool = [];
+  const ee = new EE();
+
+  let server = net.createServer();
+
+  server.on('connection', function (socket) {
+    let client = chatClient(socket);
+    pool.push(client);
+    socket.on('data', function (data) {
+      console.log('data received from client!');
```

Comment:
Much log, Wow. Pull those out when you're ready to submit work. Makes it cleaner to read for me. :-)